src/public/modules/core/css/avatar-dropdown.scss

```diff
@@ -0,0 +1,18 @@
+.navbar {
```

Comment:
probably best not to commit this? haha

Reply:
Oof, I knew I accidentally committed it hehe